src/Adapters/Storage/Postgres/PostgresStorageAdapter.js

```diff
@@ -2063,13 +2066,11 @@ export class PostgresStorageAdapter implements StorageAdapter {
     schema: SchemaType,
     fieldNames: string[]
   ) {
-    // Use the same name for every ensureUniqueness attempt, because postgres
```

Comment:
Here’s the main part of the bug. The assumption that was made in the comments is incorrect